cmd/fuzzbinder/main.go

```diff
@@ -50,8 +52,17 @@ func main() {
 		log.Fatalf("could not get absolute path for storing the fuzzing claim: %v", err)
 	}
 
+	// Get context
+	ctx := context.Background()
+
+	// Create new GCS client
+	client, err := gcsutil.NewClient(ctx)
```

Comment:
I suggest you wrap `ctx` in `Client`, instead of passing around as a separate argument.

Reply:
It is recommended by the Context package to not do so: 

`Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx:

func DoSomething(ctx context.Context, arg Arg) error {
    // ... use ctx ...
}`

However,